mkdocs.yml

```diff
@@ -0,0 +1,107 @@
+site_name: Aeternity JavaScript SDK
+extra_css: [extra.css]
+theme:
+    name: readthedocs
+    highlightjs: true
+    hljs_languages:
+        - yaml
+        - rust
+        - erlang
+        - javascript
+        - js
+        
+markdown_extensions:
+    - attr_list
+    - admonition
+    - codehilite
+    #- pymdownx.details
+    #- pymdownx.superfences
+
+nav:
```

Comment:
`readthedocs` theme [supports only two levels of nesting](https://github.com/mkdocs/mkdocs/issues/2010). So, I'm proposing to use a different theme or to rearrange navigation somehow.